app/views/updateCase/UpdateCaseConfirmationView.scala.html

```diff
@@ -29,10 +29,13 @@
         <h1 class="govuk-panel__title">
             @messages("updateCase.confirmation.heading")
         </h1>
+        <div class="govuk-panel__body">
+            <div id = "entryNumber">@messages("updateCase.confirmation.caseId")</div>
+            <strong>@caseId</strong>
+        </div>
```

Comment:
![image](https://user-images.githubusercontent.com/86240174/143051994-9bdbe266-3127-4d03-b1c3-e2188c6564b7.png)
there's a `br` element before the reference number in the prototype, we might want to add that as well